snforge_std/src/cheatcodes/erc20.cairo

```diff
@@ -0,0 +1,50 @@
+use starknet::{ContractAddress};
+use snforge_std::cheatcodes::storage::{map_entry_address, store};
+
+const STRK_CONTRACT_ADDRESS: felt252 =
+    0x04718f5a0fc34cc1af16a1cdee98ffb20c31f5cd61d6ab07201858f4287c938d;
+
+#[derive(Drop, Serde, Copy)]
+pub struct CustomToken {
+    pub contract_address: ContractAddress,
+    pub balances_variable_selector: felt252,
+}
+
+#[derive(Drop, Copy, Clone)]
+pub enum Token {
+    STRK,
+    Custom: CustomToken,
+}
+
+#[generate_trait]
+pub impl TokenImpl of TokenTrait {
+    fn contract_address(self: Token) -> ContractAddress {
```

Comment:
Just to confirm, we don't need to take `self: @Token` because it implements `Copy` trait right? It's the same as rust.